.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,39 @@
 <!--
-Thanks for sending a pull request!
+Thank you for contributing! Please remember to:
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
+- Read our contributing guidelines: https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests
+- Sign your commits with your real name. You can amend previous commits with `git commit -s --amend`.
+-->
+
+#### Description
+<!-- Provide a clear summary of your changes and why they are needed. -->
```

Comment:
This should be the commit message ideally. Unless you send many commits in one PR when it can be a smaller description of the full changes being made.